cluster-management/setup/cluster-architectures/index.md

```diff
@@ -138,12 +144,15 @@ Look how simple the cloud-config becomes:
 #cloud-config
 
 coreos:
+  etcd2:
+    proxy: on 
+    listen-client-urls: http://localhost:2379
+    initial-cluster: etcdserver=http://10.0.0.101:2380
   fleet:
-    # replace this IP
-    etcd_servers: "http://10.0.0.101:4001"
+    etcd_servers: "http://localhost:2379"
   units:
-    - name: etcd.service
-      mask: true
+    - name: etcd2.service
```

Comment:
yes, it will need to be etcd2. 

@crawford is it safer to mask the other etcd unit here? or should we remove?